src/Microsoft.IdentityModel.Tokens/Telemetry/ITelemetryClient.cs

```diff
@@ -9,24 +9,29 @@ internal interface ITelemetryClient
     {
         internal void LogConfigurationRetrievalDuration(
             string metadataAddress,
+            string configurationSource,
```

Comment:
This is not a binary breaking change, right? ITelemetryClient is used in other projects but only through TelemetryClient implementation.

Reply:
Good point on this and your other comment. Adding overloads to minimize the risk here.